Scala/syntax_test_scala.scala

```diff
@@ -550,6 +551,7 @@ type Foo = Bar[A] forSome { type A }
 =>     // this is here to act as a random terminator to the above partial syntax
 
    case class Thingy(abc: Int) extends Other
+// ^^^^^^^^^^^^^^^^^ meta.class.identifier.scala
```

Comment:
I agree that seems quite odd.  The reason is that `case class` is a very different construct from `class`.  Different in a way that `final class` or `private class` would not be.  With that said, I'm not 100% convinced that this difference is significant enough to merit inclusion in the local symbol list.  So if you feel strongly the other way (i.e. that `case` should be excluded), I can change it.